package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "kong-dashboard",
-  "version": "3.1.0",
+  "name": "kong-dashboard-tmp",
+  "version": "0.1.199",
```

Comment:
What's going on in?
Please don't modify the version yourself, I'll take care of that after merging your PR.